hello-world.php

```diff
@@ -5,15 +5,15 @@
  *
  * @package           HelloWorld
  * @author            Micah Wood
- * @copyright         Copyright 2022 by Micah Wood - All rights reserved.
+ * @copyright         Copyright 2023 by Micah Wood - All rights reserved.
```

Comment:
@wpscholar Are we sure this is the right year? You've changed this a few times...